README.rst

```diff
@@ -144,17 +144,17 @@ connect to your chosen backend (XMPP, Hipchat, Slack ...) server.
 The first time you start Err, it is recommended to run it in foreground mode. This can
 be done with::
 
-    <path_to_install_directory>/scripts/err.py
+    <path_to_install_directory>/errbot/err.py
```

Comment:
@gbin I wanted to keep the changes to `README` at the bare minimum, but if you want I can rewrite this  whole section so that `errbot` will be the only suggested option.